<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RandomSeedForm_template</class>
 <widget class="QDialog" name="RandomSeedForm_template">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string extracomment="The name of the random seed dialog">Random seed</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="component1GroupBox">
     <property name="title">
      <string notr="true">Component 1 (more details will be written from the program)</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="2">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="s12Label">
          <property name="text">
           <string notr="true">s12</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="s12LineEdit"/>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="s10Label">
          <property name="text">
           <string notr="true">s10</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="s10LineEdit"/>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="s11Label">
          <property name="text">
           <string notr="true">s11</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="s11LineEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>6</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="component2GroupBox">
     <property name="title">
      <string notr="true">Component 2 (more details will be written from the program)</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="2">
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="s22Label">
          <property name="text">
           <string notr="true">s22</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="s22LineEdit"/>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QLabel" name="s20Label">
          <property name="text">
           <string notr="true">s20</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="s20LineEdit"/>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="s21Label">
          <property name="text">
           <string notr="true">s21</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="s21LineEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>7</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>81</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="helpPushButton">
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okPushButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>s10LineEdit</tabstop>
  <tabstop>s11LineEdit</tabstop>
  <tabstop>s12LineEdit</tabstop>
  <tabstop>s20LineEdit</tabstop>
  <tabstop>s21LineEdit</tabstop>
  <tabstop>s22LineEdit</tabstop>
  <tabstop>helpPushButton</tabstop>
  <tabstop>okPushButton</tabstop>
  <tabstop>cancelPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
